lib/membrane_h264_format/remote.ex

```diff
@@ -1,14 +1,19 @@
 defmodule Membrane.H264.RemoteStream do
   @moduledoc """
-  Format definition for H264 video streams with decoder configuration transmitted out of band.
+  Format definition for packetized, remote H264 video streams.
 
-  Example of such a stream is H264 depayloaded from a container like MP4 or FLV, where decoder configuration is signalled outside of the H264 bytestream.
+  Examples of such a stream:
+  * H264 depayloaded from a container like FLV, where
+  decoder configuration is signalled outside of the H264 bytestream.
+  * H264 depayloaded from an RTP stream which is always aligned to
+  NAL units.
   """
-  @enforce_keys [:decoder_configuration_record, :stream_format]
+  @enforce_keys [:alignment, :decoder_configuration_record, :nalu_format]
```

Comment:
The decoder configuration record should rather default to `nil`. Actually, I'd consider making all fields optional

Reply:
I think that at least `alignment` should be present. If alignment is unknown, then `Membrane.RemoteStream` with `type: :bytestream` should be used. We might consider making `nalu_format` `nil` or `:unknown`

Reply:
These rules are getting complicated. I would personally prefer making all fields optional but, and this is a huge one, we would probably need to get rid of `%RemoteStream{content_format: H264}` to avoid confusion and maybe end up completely redoing `Membrane.RemoteStream` format in the process.

Reply:
Consider Membrane.File plugin use case